### DISCUSSION TITLE
Don't work if Thunderbird is closed and started new

### DISCUSSION THREAD

**Discussion by wedeen** [Extended-Thunder/send-later#591] [General]
Hello,
Unfortunately the app doesn't work anymore, I have Thunderbird 102.14.0 (64-bit)
With a newly created profile it works as long as you never close Thunderbird. Closing and reopening once is enough and the planned messages are no longer sent. Does anyone else have the error?
Greetings Verena

**Comment by jikamens** [comment]
What OS are you on?

When you say "close" do you mean minimize the Thunderbird window or actually closing it?

If you're on macOS, note that if you close (not minimize) all of your Thunderbird windows, then Send Later won't deliver messages, even if macOS still thinks the Thunderbird application is running (i.e., it's showing a dot in the dock). Closing all of the windows is equivalent to quitting from Thunderbird entirely in this regard.

Are you using an IMAP server or an Exchange server with OWL?

Are you saying that if you close and reopen the Thunderbird app, the overdue messages aren't sent even after you've reopened the app? How long are you waiting after reopening the app to see if the overdue messages will get sent?

Do you have Thunderbird's preferences configured not to send overdue messages? Could that be why the overdue messages are not being sent when you reopen Thunderbird?

Is Send Later displaying any error pop-ups?

**Comment by jikamens** [comment]
Closed for no response.
